### PR TITLE
feat: display account details (wip)

### LINKED ISSUE #32: Display Account in the navbar

### PR DESCRIPTION
**Pull request by JustinCast**
# Display account details
_Display account details along with a new functionality to store the five last tx to the localStorage. So the user can see them_
## How should this be tested?
1. `yarn start`
## Notes or observations
_no observations_
## Linked issues
closes #32

### CODE REVIEW

**Comment by kesar** [comment]
its looking very good. make sure you take in consideration mobile for the address thingy.

also perhaps we can remove the Logout ETH button and put it inside of the dialog. So to logout on eth, you click in the address and in the new dialog you have the button 👍🏻

**Comment by JustinCast** [comment]
> its looking very good. make sure you take in consideration mobile for the address thingy.
> 
> also perhaps we can remove the Logout ETH button and put it inside of the dialog. So to logout on eth, you click in the address and in the new dialog you have the button 👍🏻

Nice catch. Yep, locate the logout button inside the dialog would be nice 👍

File: src/utils/EthDataProvider/EthDataProvider.js [hunk 1]
```diff
@@ -192,7 +192,9 @@ const increaseAmount = async (amount, wallet, handleConfirmation) => {
       .then(() => handleConfirmation("success"))
       .catch(err => handleConfirmation(err));
 
-    return result;
+    console.log(hashes);
```
Comment (kesar) [inline src/utils/EthDataProvider/EthDataProvider.js:195]:
```suggestion
```

**Review by kesar — APPROVED** [review]

File: src/components/ui/ethereumWalletModal.js [hunk 2]
```diff
@@ -24,16 +25,24 @@ const StyledWalletSvg = styled(Card.Img)`
...
     wallet.reset();
     wallet.connect(connector);
   };
+
+  const handleWalletButtonClick = value => {
+    activate(value);
+    localStorage.setItem("__WALLET_CONNECTED", JSON.stringify(value));
+    setShow(false);
+  };
+
```
Comment (kesar) [inline src/components/ui/ethereumWalletModal.js:41]: beautiful refactoring 👍🏻 

https://deviq.com/principles/boy-scout-rule

**Review by kesar — APPROVED** [review]

File: src/components/ui/ethereumWalletModal.js (hunk 2 above)
Comment (JustinCast) [reply src/components/ui/ethereumWalletModal.js:41]: Great resource!

**Pull request merged**